src/js/components/Drop/__tests__/Drop-test.js

```diff
@@ -187,6 +187,28 @@ describe('Drop', () => {
     expect(actualRoot).toBe(document.body);
   });
 
+  test(`should render correct margin for position target`, () => {
```

Comment:
Wondering if we can enhance Drop-test to be like layer where it holds an array of all positions and runs this test `forEach` of them to ensure all align combos are tested.

Reply:
good idea I didnt think of that.. probably would of helped instead of manually lol